src/main/webapp/app/entities/feedback.model.ts

```diff
@@ -25,7 +25,7 @@ export class Feedback implements BaseEntity {
     public gradingInstruction?: GradingInstruction;
     public text?: string;
     public detailText?: string;
-    public reference?: string;
+    public reference?: string | null;
```

Comment:
I guess you shouldn´t use `null` as type annotation here since @krusche removed that in [#2135](https://github.com/ls1intum/Artemis/pull/2135). In TypeScript rather use `undefined` which is implied by the question mark.

Reply:
Done